vars/addRcDetailsComment.groovy

```diff
@@ -77,6 +77,7 @@ void call(Map args = [:]) {
 }
 
 def getDockerScanResult(String component, def distributionRcBuildNumber) {
+    withCredentials([usernamePassword(credentialsId: 'jenkins-github-bot-token', passwordVariable: 'GITHUB_TOKEN', usernameVariable: 'GITHUB_USER')]) {
```

Comment:
Nit, the spacing for the curly braces needs to be corrected.
Hard to read.

Thanks.

Reply:
Fixed the formatting. Also handled the error while fetching docker-scan results. It should not error out now.